### ISSUE TITLE
Icon Request: Docker Compose

### ISSUE THREAD

**Issue by echoaman** [simple-icons/simple-icons#5725]
**Brand Name:** Docker Compose
**Website:** https://docs.docker.com/compose/
**Alexa rank:** 173.4 k (the site returned stats of docker.com)
**Github Stars:** ~ 22.6k
  

**Official resources for icon and color:** https://github.com/docker/compose/blob/master/logo.png

(Sorry, I was unable to find a SVG image)

**Comment by PeterShaggyNoble** [comment]
Thanks for the request, @echoaman.

It looks like the icon for Compose actually differs significantly from their logo; I found [this SVG](https://github.com/docker/docker.github.io/blob/master/images/compose_48.svg) while digging through their repos. Being more familiar with the brand, would you agree with that?

Also going to ping @mdlinville who added that SVG for their feedback as to which we should use and, if it's the full octopus logo, to see if they can help us out with or point us in the right direction of a vector version of it.

**Comment by echoaman** [comment]
Hi @PeterShaggyNoble .

I have never seen that SVG before. I have never heard/seen anyone associate docker compose with that SVG. I believe that the octopus logo is the official one. 

That being said, I could have missed something. I guess taking feedback from @mdlinville would be the right thing to do.

**Comment by mdlinville** [comment]
Sorry folks, I haven't worked for Docker since 2018 and have no useful input.

**Comment by cloudagon** [comment]
I have not seen the `compose_48.svg` either. In [the actual README](https://github.com/docker/compose#docker-compose) they use the octopus logo. In the [docker-compose Chocolatey package](https://community.chocolatey.org/packages/docker-compose#individual), they use the octopus icon too. It is written by docker contributors and the repository is owned by a docker engineer.

**Comment by PeterShaggyNoble** [comment]
Thanks, @mdlinville & @cloudagon 👍🏻 Seems pretty clear, so, that the octopus logo is what we should run with.

Going to ping @thaJeztah of the @docker team as the current top contributor to [their docs repo](https://github.com/docker/docker.github.io) to see if they can help us out in sourcing a vector version.

**Comment by thaJeztah** [comment]
Yeah, I don't think that svg should be used. I think it was used at some point in some schematics (and some redesign of the website to stylize somethings). There's also the "compose spec" logo (which is a bit more stylised); https://github.com/compose-spec


let me /cc @chris-crone and @ndeloof, who may have input on this

**Comment by ndeloof** [comment]
I don't know who created the compose-spec logo, I guess the original design was vector-based and could be imported - maybe @nebuk89 knows? But this is compose-spec, not docker-compose

I also agree with Sebastiaan svg should not be used, this is not relevant for the compose project. I don't think compose logo has ever been converted into vector-graphic.

**Comment by PeterShaggyNoble** [comment]
Thanks for taking the time to respond, @thaJeztah & @ndeloof, it's much appreciated 🙂

OK, so, that SVG I found is definitely out! 😆 For any of our contributors who want to take a stab at manually vectorising the full logo, I found a larger version [here](https://github.com/docker/docker.github.io/blob/master/images/Compose.png) which may be easier to work with.

**Comment by PeterShaggyNoble** [comment]
Excellent 🙂 We really appreciate that, @ndeloof.

For the monochrome treatment, would you be happy if we just used the outlines? Something along the lines of:

![](https://user-images.githubusercontent.com/15157491/118829880-6dc33380-b8b6-11eb-8c06-1acd926e0e2c.png)

**Comment by PeterShaggyNoble** [comment]
SVG is now at https://github.com/docker/compose/blob/v1/logo.svg
